packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -97,6 +97,9 @@ module.exports = strapi => {
             context: ctx,
           };
         },
+        formatError: err => {
```

Comment:
Can you instead pass strapi.plugins.graphql.config.formatError so a user can define it in the config/plugins.js instead of using extensions. This will also avoid creating a dumb formatError function file

Reply:
You're right. I didn't know there was a setting for the user to pass on the function. I fixed it.